recipes/nysdec_state_facility_permits/build.py

```diff
@@ -116,6 +117,16 @@ def _geocode(df: pd.DataFrame) -> pd.DataFrame:
     )
     return df
 
+def correct_coords(df):
+    corr = pd.read_csv("../_data/air_corr.csv", dtype=str, engine="c")
```

Comment:
It's unclear why we need to read in air_corr.csv again.  Is there a way to use the filtered correction file corr_dict declared above?